app/javascript/common/http.js

```diff
@@ -0,0 +1,17 @@
+const http = window.http; // eslint-disable-line prefer-destructuring
```

Comment:
this can be written as `const { http } = window;` for es6 destructuring (does the same thing)

Reply:
ok, fixing in API.js as well :)